OpenRA.Mods.Common/Scripting/Global/ActorGlobal.cs

```diff
@@ -41,11 +41,18 @@ public Actor Create(string type, bool addToWorld, LuaTable initTable)
 					if (initType == null)
 						throw new LuaException("Unknown initializer type '{0}'".F(typeName));
 
-					// Cast it up to an IActorInit<T>
-					var genericType = initType.GetInterfaces()
-						.First(x => x.IsGenericType && x.GetGenericTypeDefinition() == typeof(IActorInit<>));
-					var innerType = genericType.GetGenericArguments().First();
-					var valueType = innerType.IsEnum ? Enum.GetUnderlyingType(innerType) : innerType;
+					// HACK: Handle OwnerInit as a special case until ActorInit creation can be rewritten
```

Comment:
This is the key thing that lets us split one ugly +/- ~700 line PR into two ~350 line PRs to make reviewing a bit easier.